src/openforms/submissions/utils.py

```diff
@@ -71,8 +79,46 @@ def remove_submission_uploads_from_session(
         remove_upload_from_session(attachment.temporary_file, session)
 
 
+def get_confirmation_email_components(submission: Submission):
+    if (
+        submission.form.confirmation_email_option
+        == ConfirmationEmailOptions.form_specific_email
+    ):
+        email_template = submission.form.confirmation_email_template
+        subject = email_template.subject
```

Comment:
isn't this supposed to be a template too? Global config is a template, but form override then isn't? That violates the end-user principle of least surprise.

This entire function should just do the template resolving, and pass subject, html and text templates to render in the other function.